R/fetch_description.R

```diff
@@ -168,3 +168,111 @@ fetch_description <-
 
     return(output)
   }
+
+
+#'Download and save a description of a survey in Qualtrics Survey Format (QSF)
+#'
+#'@param surveyID String. Unique ID for the survey you want to download.
+#'  Returned as "id" by the [all_surveys] function.
+#'@param formatting String. Adjustments to the downloaded QSF JSON to be either
+#'  more human-readable or more compact.  Options are "none" (default, no
+#'  adjustment), "pretty" (more human-readable, using
+#'  \code{\link[jsonlite]{prettify}}), or "compact" (more compact, using
+#'  \code{\link[jsonlite]{minify}})
+#'@param file String. Desired filename, including path if desired.
+#'  Typical filename extension is .qsf, though not required (and will not be
+#'  added if absent).  Argument ignored if \code{save == FALSE}.
+#'@param save Boolean. Should downloaded QSF be saved to a file (\code{TRUE},
+#'  default), or returned as output (\code{FALSE})?  Even when \code{TRUE},
+#'  downloaded JSON is returned invisibly.
+#'
+#'
```

Comment:
Can we add an example? It would be a nice way to re-emphasize how to save as a file vs. only return the content. It will likely need to be wrapped so it doesn't execute like basically all our other examples.